### PR TITLE
Patching build scripts for new blob storage URLs

### PR DESCRIPTION
**Pull request by shiftylogic**
The URLs for pulling down .NET Core installs changed for 2.1. This is PR is preparation for that.

NOTE: This is not strictly necessary right now, but will become necessary to get PR #1554 to pass CI once the CLI issues are fixed.

### CODE REVIEW

**Comment by shiftylogic** [comment]
FYI, the install-dotnet.(sh|ps1) are not changes that I made. I just updated them from the scripts in the [CLI repo](https://github.com/dotnet/cli/tree/master/scripts/obtain).

**Comment by shiftylogic** [comment]
@dotnet-bot test Innerloop Ubuntu16.04 Release Build and Test

**Comment by shiftylogic** [comment]
@ahsonkhan @joshfree @KrzysztofCwalina

File: build.sh [hunk 1]
```diff
@@ -32,10 +32,10 @@ echo "BuildVersion=$BuildVersion."
 
 if [ ! -d "dotnet" ]; then
   echo "dotnet.exe not installed, downloading and installing."
-  if [ "$Version" = "<default>" ]; then 
+  if [ "$Version" = "<default>" ]; then
     Version=$(head -n 1 "DotnetCLIVersion.txt")
   fi
-  ./scripts/install-dotnet.sh -Version "$Version" -InstallDir "dotnet"
+  ./scripts/install-dotnet.sh -Channel master -Version "$Version" -InstallDir "dotnet"
```
Comment (ahsonkhan) [inline build.sh]: Does master need to be in quotes?

Comment (shiftylogic) [inline build.sh]: It doesn't need to me. If it were a string containing a space it would matter

**Pull request merged**